.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -15,6 +15,16 @@ Please fill in as much of the template below as you're able. Feel free to delete
 ### Search terms you've used
 <!-- What search terms have you used to check whether this bug was already reported? -->
 
+### Impacted package
+
+What package(s) of the monorepo is impacted by the bug ?
+
+- [ ] solid-client-authn-browser
+- [ ] solid-client-authn-node
+- [ ] solid-client-authn-core
+- [ ] oidc-client-ext
+- [ ] Other (please specify): ...
```

Comment:
Sounds like this might be useful :) (Though maybe just "is your app running in Node or in the browser" would be easier to answer and provide most of the relevant info.)

```suggestion
- [ ] Other (please specify): ...
- [ ] I don't know
```

Reply:
That's a good point, actually I'll change to that, in 90% of the cases that's going to be what we need to know, and I expect the remaining 10% to be bugs we report ourselves ^^